### ISSUE TITLE
java.lang.RuntimeException: Unable to start activity 

### ISSUE THREAD

**Issue by sshnakamoto** [akshay2211/PixImagePicker#7]
Do I need to ask permission before? If yes, why don't you have specified on repo? Your demo app don't requires so.

I'm very begginer on Android, could you help me with?

```c#
05-31 11:55:48.369 4045-4045/com.example.android.test E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.example.android.test, PID: 4045
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.example.android.test/com.fxn.pix.Pix}: java.lang.SecurityException: Permission Denial: reading com.android.providers.media.MediaProvider uri content://media/external/images/media from pid=4045, uid=10114 requires android.permission.READ_EXTERNAL_STORAGE, or grantUriPermission()
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2779)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2857)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1590)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6499)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.SecurityException: Permission Denial: reading com.android.providers.media.MediaProvider uri content://media/external/images/media from pid=4045, uid=10114 requires android.permission.READ_EXTERNAL_STORAGE, or grantUriPermission()
        at android.os.Parcel.readException(Parcel.java:2004)
        at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:183)
        at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:135)
        at android.content.ContentProviderProxy.query(ContentProviderNative.java:418)
        at android.content.ContentResolver.query(ContentResolver.java:754)
        at android.content.ContentResolver.query(ContentResolver.java:704)
        at android.content.ContentResolver.query(ContentResolver.java:662)
        at com.fxn.utility.Utility.getCursor(Utility.java:163)
        at com.fxn.pix.Pix.updateImages(Pix.java:537)
        at com.fxn.pix.Pix.initialize(Pix.java:532)
        at com.fxn.pix.Pix.onCreate(Pix.java:316)
        at android.app.Activity.performCreate(Activity.java:7044)
        at android.app.Activity.performCreate(Activity.java:7035)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2732)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2857) 
        at android.app.ActivityThread.-wrap11(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1590) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6499) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
```

**Comment by sshnakamoto** [comment]
this bug is caused by calling the following method: `Pix.start(this, 10)`. When it's the first time, Pix does not ask about storage permissions. So that, throw above error.

**Comment by sshnakamoto** [comment]
Video demo error when calling `Pix.start(this, 10)`
https://www.youtube.com/watch?v=DjE2SktVz8w

**Comment by sshnakamoto** [comment]
Video demo showing no error when specifying third argument `Pix.start(this, 10, 1)`
https://www.youtube.com/watch?v=Qs9YG51iOFA

**Comment by akshay2211** [comment]
Hello @sshnakamoto,
Thank you for reporting this bug.
it is been fixed in the latest update (_version_ **1.0.5**)
